tests/unit/test_shard_aware.py

```diff
@@ -0,0 +1,38 @@
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest  # noqa
+
+from cassandra.connection import ShardingInfo
+from cassandra.metadata import Murmur3Token
+
+class TestShardAware(unittest.TestCase):
+    def test_parsing_and_calculating_shard_id(self):
```

Comment:
The same here: some human readable description please. See examples in (at least mine) dtests.
For instance in hintedhandoff_additional_tests.py.